test/handleActions.test.js

```diff
@@ -529,3 +529,54 @@ test('works with nested reducerMap and identity handlers', () => {
     message: 'hello'
   });
 });
+
+test('works with combineActions nested', () => {
+  const {
+    app: { counter, sameCounter }
+  } = createActions({
+    APP: {
+      COUNTER: undefined,
+      SAME_COUNTER: undefined
+    }
+  });
+  const { app } = createActions({
+    APP: {
+      COUNTER: {
+        INCREMENT: amount => ({ amount }),
+        DECREMENT: amount => ({ amount: -amount })
+      },
+      SAME_COUNTER: {
+        INCREMENT: amount => ({ amount }),
+        DECREMENT: amount => ({ amount: -amount })
+      }
+    }
+  });
```

Comment:
Alright, but still couldn't you just remove L536 and do this instead?

```js

const { app } = createActions({
  APP: {
    COUNTER: {
      INCREMENT: amount => ({ amount }),
      DECREMENT: amount => ({ amount: -amount })
    },
    SAME_COUNTER: {
      INCREMENT: amount => ({ amount }),
      DECREMENT: amount => ({ amount: -amount })
    }
  }
});

const { counter, sameCounter } = app
```

Reply:
@timche This doesn't work because of line L558: `combineActions(counter, sameCounter)`. `combineActions` expect actions, not keys. I can rewrite L536 with:
```js
const app: { counter, sameCounter } = createActions('APP/COUNTER', 'APP/SAME_COUNTER')
```
..if you prefer.
Or totally change action names and do something like `API_CALL` and `API_CALL_SUCCESS`?

I know this is not very clear but the initials dispatched (and combined) actions have to be different than the one nested in the combine.

Reply:
Sorry, didn't have much time to look into that and try it out myself. I just want it to be changed so the code makes sense.

Reply:
It's totally documented: https://redux-actions.js.org/api-reference/createaction-s#createaction-type-payloadcreator and also used in an example below
```
SET: undefined // given undefined, the identity function will be used
```
But if you prefer I can write `createAction('API_CALL_1/LOADING')`. This way, I use explicitly the `/` separator which is implicit in the object form.

Reply:
I know that `undefined` will use the identity function instead, but I'm more concerned about using `createActions` twice and the first one is just being used for creating action types and the second to create action creators which is overall looks quite confusing imo.

Reply:
I know.. As I said, in a real app, the `LOADING` action would have been created else where (in a middleware for example) so it won't be confusing.

However, this is a simple test ensuring the fix works, maybe we can pass through and improve it later?

I've changed it a few times and I don't have many options left. If you propose me something, I would be happy to include it, I'm out of ideas.

Reply:
Yeah, I think this is too much nitpicking from my side, sorry. After all, this issue is fixed. Thanks!